selfdrive/can/parser.cc

```diff
@@ -55,7 +55,6 @@ bool MessageState::parse(uint64_t sec, uint16_t ts_, uint8_t * dat) {
       }
     } else if (sig.type == SignalType::VOLKSWAGEN_COUNTER) {
         if (!update_counter_generic(tmp, sig.b2)) {
-        INFO("0x%X CHECKSUM FAIL\n", address);
```

Comment:
why not COUNTER FAIL?

Reply:
Because all COUNTER fail messages (Honda, Volkswagen, Pedal) are generated in update_counter_generic already, parser.cc:86.